libs/utils/src/logging.rs

```diff
@@ -178,6 +202,29 @@ pub fn init(
     Ok(())
 }
 
+pub fn init_otel_tracing(otel_enablement: OtelEnablement) -> anyhow::Result<Option<OtelGuard>> {
+    let otel_subscriber = match otel_enablement {
+        OtelEnablement::Disabled => None,
+        OtelEnablement::Enabled {
+            service_name,
+            export_config,
+            runtime,
+        } => {
+            let otel_layer = runtime
+                .block_on(tracing_utils::init_tracing(&service_name, export_config))
+                .with_filter(LevelFilter::INFO);
```

Comment:
Why is this hard-coded? You can use a `tracing_subscriber::EnvFilter` with a different env var name.

Reply:
Because all perf traces are at info level (see https://github.com/neondatabase/neon/pull/11140). I agree this is a bit implicit. Open to better ideas.

Reply:
Don't have any, maybe a comment would be sufficient.